extensions/scarb-snforge-test-collector/src/metadata.rs

```diff
@@ -0,0 +1,112 @@
+use anyhow::{anyhow, Context, Result};
+use cairo_lang_filesystem::db::Edition;
+use cairo_lang_project::{AllCratesConfig, SingleCrateConfig};
+use cairo_lang_utils::ordered_hash_map::OrderedHashMap;
+use camino::{Utf8Path, Utf8PathBuf};
+use scarb_metadata::{CompilationUnitMetadata, Metadata, PackageMetadata};
+use smol_str::SmolStr;
+use std::path::PathBuf;
+
+/// Represents a dependency of a Cairo project
+#[derive(Debug, Clone)]
+pub struct LinkedLibrary {
```

Comment:
Umm, this would probably translate to compilation unit components, but since you are dropping scarb project abstraction I am ok with this struct having a different name

Reply:
For me the value here is, we are going to use the same name when discussing thins between teams :p

Reply:
I see your point and it is generally true. However, this is just a subset of component metadata, and a very primitive one. In fact, when looking at usages I would be in favour of dropping the struct completely - it could be passed as a plain tuple without any damage to readability.

Reply:
Ok, your call